data/net/minecraft/world/entity/monster/Spider.mapping

```diff
@@ -17,12 +17,12 @@ CLASS net/minecraft/world/entity/monster/Spider
 		ARG 1 pose
 		ARG 2 size
 	METHOD isClimbing ()Z
-		COMMENT Returns true if the WatchableObject (Byte) is 0x01 otherwise returns false. The WatchableObject is updated using setBesideClimbableBlock.
+		COMMENT Returns {@code true} if the WatchableObject (Byte) is 0x01 otherwise returns false. The WatchableObject is updated using setBesideClimbableBlock.
```

Comment:
```suggestion
		COMMENT Returns {@code true} if the WatchableObject (Byte) is 0x01 otherwise returns {@code false}. The WatchableObject is updated using setBesideClimbableBlock.
```